pkg/aws/cloud.go

```diff
@@ -52,8 +53,19 @@ func NewCloud(cfg CloudConfig, metricsRegisterer prometheus.Registerer) (Cloud,
 		cfg.Region = region
 	}
 
-	awsCfg := aws.NewConfig().WithRegion(cfg.Region).WithSTSRegionalEndpoint(endpoints.RegionalSTSEndpoint)
+	awsCfgAPPMESH := &aws.Config{
```

Comment:
Variables in Go should generally be [camel case](https://en.wikipedia.org/wiki/Camel_case); no reason for "appmesh" to be entirely caps here.